### DISCUSSION TITLE
p2.2 of UIA vs Scarlet forces update

### DISCUSSION THREAD

**Discussion by Nrmot** [Nrmot-GithubRepo/Project-CBT#18] [Project CBT: Latest Dev Releases]
changelog:

few more quests got greggified

new names for Blast Furnace and Macerator:

Will plan to rename all of GT machines based on tier:

Here's an example of what it would look like, using Electric Furnace as an example. If the word electric is present then the word electric will be removed at UV level which is present here:

ULV - Ungreggy Electric Furnace
LV - Loser's Electric Furnace
MV - Basic Electric Furnace
HV - Advanced Electric Furnace
EV - Superadvanced Electric Furnace
IV - Accelerated Electric Furnace
LuV - Superaccelerated Electric Furnace
ZPM - Hyperaccelerated Electric Furnace
UV - Quantum Blast Furnace
UHV - Accelerated Quantum Furnace
UEV - Hyperaccelerated Quantum Furnace
UIV - Cosmic Furnace
UXV - X-Furnace
OpV - CBT Furnace

Heres an example except Macerator is used instead
ULV - Ungreggy Macerator
LV - Loser's Macerator
MV - Basic Macerator
HV - Advanced Macerator
EV - Superadvanced Macerator
IV - Accelerated Macerator
LuV - Superaccelerated Macerator
ZPM - Hyperaccelerated Macerator
UV - Quantum Macerator
UHV - Accelerated Macerator
UEV - Hyperaccelerated Macerator
UIV - Cosmic Macerator
UXV - X-Macerator
OpV - CBT Macerator

still has client only and NO server files yet

<hr /><em>This discussion was created from the release <a href='https://github.com/Nrmot-GithubRepo/Project-CBT/releases/tag/v0.0.0-dev9'>p2.2 of UIA vs Scarlet forces update</a>.</em>

**Comment by sniperfoxeh** [comment]
damn it i was 36 mins late

**Comment by sniperfoxeh** [comment]
i have spent the last 15 minuets tryna install this

**Reply by Nrmot** [reply]
damn

[client.zip then import to multimc, prismlauncher or curseforge app](https://github.com/Nrmot-GithubRepo/Project-CBT/releases/download/v0.0.0-dev9/Project.CBT-DEV.zip)
